doc/ref_model/chapters/chapter04.md

```diff
@@ -90,26 +90,20 @@ This section describes a set of explicit Cloud Infrastructure capabilities and p
-| e.cap.006 | CPU core pinning support                  | Yes/No | Indicates if Cloud Infrastructure supports CPU core pinning |
-| e.cap.007 | NUMA support                              | Yes/No | Indicates if Cloud Infrastructure supports NUMA             |
+| e.cap.006 | CPU pinning                               | Yes/No | Indicates if Cloud Infrastructure supports CPU pinning      |
+| e.cap.007 | NUMA alignment                            | Yes/No | Indicates if Cloud Infrastructure supports NUMA             |
 | e.cap.008 | IPSec Acceleration                        | Yes/No | IPSec Acceleration                                          |
 | e.cap.009 | Crypto Acceleration                       | Yes/No | Crypto Acceleration                                         |
 | e.cap.010 | Transcoding Acceleration                  | Yes/No | Transcoding Acceleration                                    |
 | e.cap.011 | Programmable Acceleration                 | Yes/No | Programmable Acceleration                                   |
-| e.cap.012 | Enhanced Cache Management <sup>1)</sup>   | Yes/No | If supported, L=Lean; E=Equal; X=eXpanded                   |
-| e.cap.013 | SR-IOV over PCI-PT <sup>2)</sup>          | Yes/No | Traditional SR-IOV                                          |
-| e.cap.014 | GPU/NPU <sup>2)</sup>                     | Yes/No | Hardware coprocessor                                        |
-| e.cap.015 | SmartNIC <sup>2)3)</sup>                  | Yes/No | Network Acceleration                                        |
-| e.cap.016 | FPGA/other Acceleration H/W <sup>2)</sup> | Yes/No | Non-specific hardware                                       |
+| e.cap.012 | Enhanced Cache Management                 | Yes/No | If supported, L=Lean; E=Equal; X=eXpanded.  L and X cache policies require CPU pinning to be active. |
+| e.cap.013 | SR-IOV over PCI-PT                        | Yes/No | Traditional SR-IOV. These Capabilities generally require hardware-dependent drivers be injected into workloads, which is prohibited by CNTT principles. As such, use of these features shall be governed by the applicable CNTT policy. Please consult the RM Appendix for the usage policy relevant to any needed hardware Capability of this type.  |
+| e.cap.014 | GPU/NPU                                   | Yes/No | Hardware coprocessor. These Capabilities generally require hardware-dependent drivers be injected into workloads, which is prohibited by CNTT principles. As such, use of these features shall be governed by the applicable CNTT policy. Please consult the RM Appendix for the usage policy relevant to any needed hardware Capability of this type. |
+| e.cap.015 | SmartNIC                                  | Yes/No | Network Acceleration. SmartNICs that do not utilise PCI-PT are not subject to the CNTT principles, nor any related policies or prohibitions. |
```

Comment:
```suggestion
| e.cap.015 | SmartNIC                                  | Yes/No | Network Acceleration.  |
```

Reply:
@CsatariGergely The text is from a note that existed below the table and as per GSMA should be included within the text.